### DISCUSSION TITLE
What are the speech timestamps are they frame numbers or milli seconds?

### DISCUSSION THREAD

**Discussion by saumyaborwankar** [snakers4/silero-vad#25] [Q&A] [answered]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

**Comment by snakers4** [comment, marked as answer]
Hi, 

If you are testing the VAD then it's frames
Please see the examples in [colab](https://colab.research.google.com/github/snakers4/silero-vad/blob/master/silero-vad.ipynb) 
There are some examples where audio is cut
